configs/sandpack-contents/hooks/use-range-slider.js

```diff
@@ -24,107 +33,136 @@ export default function App({ min, max, stepToNumber, stepToIndex, stepByNumber,
-                        bgColor='teal.500'
-                        borderRadius='full'
-                        {...getInnerTrackProps()}
-                    />
-                </Box>
-                <Thumb 
-                    value={state.value[0]} 
-                    thumbIndex={0} 
-                    thumbProps={thumbPropsFirstIndex} 
-                    onKeyDownStepBy={onKeyDownStepBy}
-                    bgColor='teal.500'
-                />
-                <Thumb 
-                    value={state.value[1]} 
-                    thumbIndex={1} 
-                    thumbProps={thumbPropsSecondIndex} 
-                    onKeyDownStepBy={onKeyDownStepBy}
-                    bgColor='teal.500'
-                />
-            </chakra.div>
+                  {value}
+                </Badge>
+              )
+            })}
+            <Box
+              h='7px'
+              bgColor='teal.100'
+              borderRadius='full'
+              {...getTrackProps()}
+            >
+              <Box
+                h='7px'
+                bgColor='teal.500'
+                borderRadius='full'
+                {...getInnerTrackProps()}
+              />
+            </Box>
+            <Thumb
+              value={state.value[0]}
+              thumbIndex={0}
+              thumbProps={thumbPropsFirstIndex}
+              onKeyDownStepBy={onKeyDownStepBy}
+              bgColor='teal.500'
+            />
+            <Thumb
+              value={state.value[1]}
+              thumbIndex={1}
+              thumbProps={thumbPropsSecondIndex}
+              onKeyDownStepBy={onKeyDownStepBy}
+              bgColor='teal.500'
+            />
+          </chakra.div>
         </Box>
-    );
-};`,
-    Index: `import { ChakraProvider } from "@chakra-ui/react";
-import * as React from "react";
-import { createRoot } from "react-dom/client";
-import App from "./App";
-const rootElement = document.getElementById("root");
-const root = createRoot(rootElement);
-
-root.render(
-    <React.StrictMode>
+      )
+    }
+    `,
+    Index: `import { ChakraProvider } from '@chakra-ui/react'
+    import * as React from 'react'
+    import { createRoot } from 'react-dom/client'
+    import App from './App'
+    const rootElement = document.getElementById('root')
+    const root = createRoot(rootElement!)
+    
+    root.render(
+      <React.StrictMode>
         <ChakraProvider>
-            <App min={25} max={75} stepToNumber={55} stepToIndex={1} stepByNumber={10} defaultValue={[25, 75]} aria-label={['min', 'max']} />
+          <App
+            min={0}
+            max={100}
+            stepToNumber={85}
+            stepToIndex={1}
+            stepByNumber={10}
+            defaultValue={[25, 75]}
+            aria-label={['min', 'max']}
+          />
         </ChakraProvider>
-    </React.StrictMode>
-);`,
+      </React.StrictMode>,
+    )    
+  `,
     Instructions: `import { Box, Text, Kbd } from "@chakra-ui/react";
 
```

Comment:
The `Instructions` component has a double-nesting of `<Text>` which generates `<p><p>..</p></p>`.
Suggest changing the outer usage to `<Text as='div'>` at L137.